Components/Alert/Alert.html

```diff
@@ -4,9 +4,23 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
```

Comment:
This is just for icons right? your CSS classes won't be overwritten by bootstrap classes right?

Reply:
Yes, my own css won't be overwritten by bootstrap css. I attached my css after bootstrap css, so it's safe.